src/Entity/JobOffer.php

```diff
@@ -98,16 +101,24 @@ class JobOffer
      * @Gedmo\Timestampable(on="update")
      *
      * @ORM\Column(type="datetime")
+     *
      */
     private $updatedAt;
 
     /**
      * @ORM\Column(type="datetime", nullable=true)
      *
-     * @Groups({"admin", "detail"})
+     * @Groups({"admin", "detail", "list"})
```

Comment:
Out of scope.

Reply:
![WhatsApp Image 2019-12-19 at 17 01 14](https://user-images.githubusercontent.com/6358755/71205909-23cdbc80-2282-11ea-8092-84194177f538.jpeg)

Edenilson is using here (public list)